src/util/pug_roles.js

```diff
@@ -0,0 +1,13 @@
+export function getRoles (member) {
```

Comment:
I think this should rather be put in utils.js

Also could you rename the function to something like `getRoleEmojis`?